iidm/iidm-converter-api/src/main/java/com/powsybl/iidm/import_/Importers.java

```diff
@@ -440,6 +483,26 @@ public static Network loadNetwork(Path file, ComputationManager computationManag
         throw new PowsyblException("Unsupported file format or invalid file.");
     }
 
+    /**
+     * Loads a network from the specified file, trying to guess its format.
+     *
+     * @param file               The file to be loaded.
+     * @param computationManager A computation manager which may be used by import post-processors
+     * @param config             The import config, in particular definition of post processors
+     * @param parameters         Import-specific parameters
+     * @param loader             Provides the list of available importers and post-processors
+     * @param reporter           The reporter used for functional logs
+     * @return                   The loaded network
+     */
+    public static Network loadNetwork(Path file, ComputationManager computationManager, ImportConfig config, Properties parameters, ImportersLoader loader, Reporter reporter) {
```

Comment:
Is it possible to share code with the one without Reporter?

Reply:
In fact, the old `Importer` API should be called when no reporter is given, as the new API is not yet implemented by all subclasses of `Importer.java`. I didn't put implementation in all the subclasses, there are in many other modules. Indeed this is still a POC so I didn't want to affect too many artifacts. Do you agree on that?